e2e/specs/st_data_editor_config.spec.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("st.data_editor supports various configuration options", () => {
+  before(() => {
+    cy.loadApp("http://localhost:3000/");
+    cy.prepForElementSnapshots();
+  });
+
+  it("shows correct tables based on configuration", () => {
```

Comment:
Would it be better to say this test is about disable configuration? I assume there will be more configurations features / parameters added so might be good to be specific here.

Reply:
The script will contain a lot more configuration option tests, but I plan to let the spec just iterate on all and compare snapshots for simplicity. Mainly also since it's likely to properly move this to playwright soon, so this is more temporary